### ISSUE TITLE
Design updates for digest emails

### ISSUE THREAD

**Issue by TylerHendrickson** [usdigitalresponse/usdr-gost#1222]
**Task:** Update the templates used when building digest notification emails to align with the  [designs](https://www.figma.com/file/HIINnuujheHbdbDesH8mKE/Notifications?node-id=17-1158) from Figma [provided by](https://github.com/usdigitalresponse/usdr-gost/issues/934#issuecomment-1473924598) @lizharman.

**DOD:** Email design for "New Grants Digest" email notification type is updated.

**Acceptance Criteria:**
- Header content updated to include [USDR logo](https://www.notion.so/usdr/USDR-Grants-Logo-1b61e6f5062a40b4b790a94099134a7a) referenced in the Figma designs
- Date at top of grants: `Grant Digest - DD MONTH YYYY`
- Update to content on **New grants**
    -   Tag: `New`
    -   Title: `[grant name]`; wrap when long
    -   Subtitle: assigned by `[first last], as [role] of [tenant]`
    -   Update UI according to designs
- `View Grant Details` on all grants should open the Grand ID tool with that grant details view opened
- `See all new grants` always visible at the bottom of the grants digest; opens the Grant ID tool to my grants dashboard

**Implementation details:**
- The "New Grants Digest" email is currently sent according to a [JS cron](https://www.npmjs.com/package/cron) schedule defined [here](https://github.com/usdigitalresponse/usdr-gost/blob/_staging/packages/server/src/index.js#L28-L34).
- The [`sendGrantDigestForAgency()` function in `packages/server/src/lib/email.js`](https://github.com/usdigitalresponse/usdr-gost/blob/0236c4738f26eaa5fa2c75343f11a2ac4c042a4e/packages/server/src/lib/email.js#L215-L252) is called per-agency and is responsible for building the email content and sending it to each recipient for that agency.
- You will likely need to set up [nodemailer](https://nodemailer.com/about/) in order to test rendering in a mail client. Setup steps are documented in our ["GMail Setup" documentation](https://github.com/usdigitalresponse/usdr-gost/blob/_staging/docs/setup-gmail.md).
- Figma designs are available here: https://www.figma.com/file/HIINnuujheHbdbDesH8mKE/Notifications?node-id=17-1158

**Subtask of:**
- #934

**Comment by TylerHendrickson** [comment]
[Logo asset](https://user-images.githubusercontent.com/1851017/231301579-1b8fc054-4482-4179-83e2-41ddc31676ab.png)
![USDR GRANTS logo_white](https://user-images.githubusercontent.com/1851017/231301579-1b8fc054-4482-4179-83e2-41ddc31676ab.png)

**Comment by ClaireValdivia** [comment]
Feedback from NJ - once Category of Funding Activity and Funding Department name are incorporated into search, this is key info they would like to see on digest

**Comment by caitlinwinner** [comment]
@ClaireValdivia I'm going to do a design pass on this.

**Comment by ClaireValdivia** [comment]
@caitlinwinner one inconsistency between this ticket and new design file is where "view grant details" goes. also seems like button text doesn't match between ticket and designs and not sure which is correct

**Comment by caitlinwinner** [comment]
ready for eng

**Comment by ClaireValdivia** [comment]
@caitlinwinner I just updated the acceptance criteria to match the designs - can you give it a quick review and confirm that it matches your understanding?

**Comment by caitlinwinner** [comment]
Yes, looks good! the only thing I didn't address was the content in the grant summary. We've heard some partners request additional bulleted info in the summary, which we could add in if you think important.

**Comment by ClaireValdivia** [comment]
Caitlin and i connected; ticket is ready for eng!

**Comment by ClaireValdivia** [comment]
@as1729 thoughts on how I can test the notification updates?

**Comment by caitlinwinner** [comment]
@TylerHendrickson  do we have control over the "From" field? If we could say USDR Federal Grant Finder instead of grants-notifications, I think that would be ideal.

Email title: Search Name

body text preview: # new grants, title of first grant...

**Comment by ClaireValdivia** [comment]
@caitlinwinner I created this issue with the items you referenced above. is it correct that these are not blockers to the current updates going live? #2353
